### PR TITLE
[Shell Parity] [Slate] Slate Flattened Border Fix

### PR DESCRIPTION
**Pull request by Cameron-Micka**
## Overview

Fixing an issue where flattened bounding boxes would be completely flattened on an axis results in a very thin bounding box border. Bounding boxes now have a display scale (tweakable) for the flattened axis.

Capture

## Changes
- Fixes: https://github.com/microsoft/MixedRealityToolkit-Unity/issues/4976

## Verification
Please see the slates contained in the HandInteractionExamples scene.

> As a reviewer, it is possible to check out this change locally by using the following
> commands (substituting {PR_ID} with the ID of this pull request):
>
> git fetch origin pull/{PR_ID}/head:name_of_local_branch
>
> git checkout name_of_local_branch

### CODE REVIEW

File: Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs [hunk 1]
```diff
@@ -185,6 +185,27 @@ public FlattenModeType FlattenAxis
                 }
             }
         }
+
+        [SerializeField]
+        [Tooltip("When an axis is flattened what value to set that axes scale to.")]
```
Comment (wiwei) [inline Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs]:
```suggestion
        [Tooltip("When an axis is flattened, what value to set that axis's scale to.")]
```

Comment (wiwei) [reply Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs]: Here and below.

Comment (Cameron-Micka) [reply Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs]: Nice catch, fixed!

**Pull request merged**